applications/application-registry.md

```diff
@@ -0,0 +1,205 @@
+
+
+### Team Org Repos
+
+- https://github.com/Rewired-one
+- https://github.com/Emanate-official
+- https://github.com/modadao
+- https://github.com/eosphere/heyfio
+
+
+### Team Member Repos
+
+- https://github.com/torbenanderson
+- https://github.com/interwebcoding
+- https://github.com/IanMendozaJaimes
+- https://github.com/tlacloc
+- https://github.com/superstar391
+- https://github.com/djkruitbosch
+
+
+### Team LinkedIn Profiles (if available)
+
+- https://www.linkedin.com/in/torbenanderson1/
+- https://www.linkedin.com/in/interwebcoding/
+- https://www.linkedin.com/in/ian-mendoza-jaimes/
+- https://www.linkedin.com/in/erick-alejandro-casanova-cort%C3%A9s-2a549010a/
+- https://www.linkedin.com/in/collin-wang-1362a51b0/
+- https://www.linkedin.com/in/djkruitbosch/
+
+
+## Development Status
+
+- No development work has started as of this submission
+- EOS Wallet+ Blue Paper
+- Grant proposal reviewed by Greymass team, pre-requisites for further development will be captured under Milstone 1
+- Grant route followed after discussion with and recommendation by Yves
+
+
+## Development Roadmap
+
+### Milestone Summary
+
+- **Total Estimated Duration:** 16 weeks 
+- **Full-Time Equivalent (FTE):**  2.4 FTE
+- **Total Costs:** 229,000 USD
+
+
+### Milestone 1 — Initiate Project & Agree on governance structure and Application Registry metadata
+
+- **Estimated duration:** 3 weeks
+- **FTE:**  0.9
+- **Costs:** 18,081 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 1. | License | MIT |
+| 2. | Project Governance setup | Team setup, governance setup, stakeholder mapping complete. |
+| 3. | Governance agreement | Agreement on how the EOS Mainnet will provide governance on how Application developers can register their applications into the Application Registry. This includes a list of validators who will ensure that apps are following the defined rules. This also defines the rules that apps have to comply with, what metrics they have to support/report and how the app can be updated or deleted. Part of the governance will also consider user input based on their experience with apps. If an app is not behaving as per guidelines, it can be reported. We propose to use a 3/5 multisig with a dedicated governing body. We suggest these are the following entities: greymass, tokenpocket, ENF, ultra and rewired. Part of this deliverable we will also define the application data standards, such as title, description, category, icon, URL, app manifest, etc. |
```

Comment:
Please include a section of this deliverable that outlines how multiple versions of the application data standards can co-exist.

Reply:
The Data Standard will be versioned and maintain backward compatibility with previous versions unless a specific breaking change gets identified or is incompatible with improvements made to the EOS blockchain. Should a fork of the standard be required, the community engagement process will commence to ask for and act on feedback. A final decision on the data standard will be with the 3/5 multisign standard custodians.  

The https://github.com/ConsumerDataStandardsAustralia/standards/issues/266 provides a working example of how community engagement would work using GitHub. FIn addition an improvement request could suggest an EOS contract that builds approvals for data standards on chain.

Reply:
This should be explicitly added to the proposal and not just as a comment on the PR